### PR TITLE
MAINT: Convert Changelog To KAC

### PR DESCRIPTION
**Pull request by RandomNoun7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,219 +1,274 @@
...
-## 2017-04-04 Supported Release 2.0.2
+## [3.0.0] - 2017-06-02
 
-### Summary
+### Fixed
 
-This is a bug fix release.
+- Explicitly close configuration files after reading ([MODULES-4678](https://tickets.puppetlabs.com/browse/MODULES-4678))
+- Use actual choco.exe instead of the shim ([MODULES-4562](https://tickets.puppetlabs.com/browse/MODULES-4562))
+- Updated puppet version compatibility for modern Puppet agents ([MODULES-4846](https://tickets.puppetlabs.com/browse/MODULES-4846))
```
Comment (clairecadman) [inline CHANGELOG.md]: Capitalize Puppet.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,219 +1,274 @@
...
 
-### Bug Fixes
+## [2.0.2] - 2017-04-04
 
-- Fix: Use two dashes when getting package version ([MODULES-4508](https://tickets.puppetlabs.com/browse/MODULES-4508))
+### Fixed
 
+- Use two dashes when getting package version ([MODULES-4508](https://tickets.puppetlabs.com/browse/MODULES-4508))
```
Comment (clairecadman) [inline CHANGELOG.md]: *the* package version?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,219 +1,274 @@
...
+- Includes community module releases up to 1.2.1 (changelog below up to 1.2.1)
+
+### Fixed
 
+- $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
+- puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
 
-# Approved Community Module Changelog - Chocolatey Team
+# Note:
+The changelog below is for changes to the Community chocolatey-chocolatey module from which this current module takes over. It is retained below for historical purposes since for a couple releases this module did track its changes and bug fixes.
```
Comment (clairecadman) [inline CHANGELOG.md]: This sentence doesn't read very well. I've reworded it below (which is how I understood it, please adjust if it's not quite right):

"The puppetlabs-chocolatey module replaces the community chocolatey-chocolatey module. We have retained its changelog below as there were a couple of releases where we tracked the new modules changes and bug fixes here"

* Community - lower case.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,219 +1,274 @@
...
+### Added
 
+- holdable ([#95](https://github.com/chocolatey/puppet-chocolatey/issues/95))
 
-##2015-11-03 Release 1.2.0
+### Fixed
 
-- Implement holdable ([#95](https://github.com/chocolatey/puppet-chocolatey/issues/95))
-- Fix - Use install unless version specified in install ([#71](https://github.com/chocolatey/puppet-chocolatey/issues/71))
+- Use install unless version specified in install ([#71](https://github.com/chocolatey/puppet-chocolatey/issues/71))
```
Comment (clairecadman) [inline CHANGELOG.md]: ...unless *the* version *is* specified..

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,219 +1,274 @@
...
+- Install Chocolatey itself / ensure Chocolatey is installed ([PUP-1691](https://tickets.puppetlabs.com/browse/PUP-1691))
+- Custom facts for chocolateyversion and choco_install_path
 
-- Fixes [#71](https://github.com/chocolatey/puppet-chocolatey/issues/71) - Allow `ensure => $version` to work with already installed packages
+## [1.0.2] - 2015-07-23
 
+### Fixed
 
-## 2015-07-01 Release 1.0.1
+- Allow `ensure => $version` to work with already installed packages [#71](https://github.com/chocolatey/puppet-chocolatey/issues/71)
```
Comment (clairecadman) [inline CHANGELOG.md]: Here you use code phrase (`), but not in other places. What is the KAC standard? Either way this should be consistent.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,219 +1,274 @@
...
+
+- Support for Windows 10
 
-## 2015-04-23 Release 0.5.2
+### Fixed
 
-- Update ReadMe
-- Add support for Windows 10.
-- Fixes [#56](https://github.com/chocolatey/puppet-chocolatey/pull/56) - Avoiding puppet returning 2 instead of 0 when there are no changes to be done.
+- Avoiding puppet returning 2 instead of 0 when there are no changes to be done [#56](https://github.com/chocolatey/puppet-chocolatey/pull/56)
```
Comment (clairecadman) [inline CHANGELOG.md]: Capitalize Puppet.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,219 +1,274 @@
...
 
+## [0.5.1] - 2015-03-31
 
-## 2015-03-31 Release 0.5.1
+### Fixed
 
-- Fixes [#54](https://github.com/chocolatey/puppet-chocolatey/issues/54) - Blocking: Linux masters throw error if module is present
+- Blocking: Linux masters throw error if module is present [#54](https://github.com/chocolatey/puppet-chocolatey/issues/54)
```
Comment (clairecadman) [inline CHANGELOG.md]: ...if *the* module is present...

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,219 +1,274 @@
...
-- Fixes #43 - check for installed packages is case sensitive
-- Fixes #18 - The OS handle's position is not what FileStream expected.
-- Fixes #52 - Document best way to pass options with spaces (#15 also related)
-- Fixes #26 - Document Chocolatey needs to be installed by other means
+- Document best way to pass options with spaces (#15 also related) - [#52](https://github.com/chocolatey/puppet-chocolatey/issues/52)
+- Document Chocolatey needs to be installed by other means - [#26](https://github.com/chocolatey/puppet-chocolatey/issues/26)
+
+### Fixed
+
+- work with newer compiled Chocolatey client (0.9.9+) - [#50](https://github.com/chocolatey/puppet-chocolatey/issues/50)
```
Comment (clairecadman) [inline CHANGELOG.md]: Keep up capitalizing the first word throughout to be consistent.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,219 +1,274 @@
...
+- check for installed packages is case sensitive - [#43](https://github.com/chocolatey/puppet-chocolatey/issues/43)
```
Comment (clairecadman) [inline CHANGELOG.md]: ...packages *that are* case sensitive...

**Comment by RandomNoun7** [comment]
@clairecadman These fixes are made in a new pull request since this one is now merged: https://github.com/puppetlabs/puppetlabs-chocolatey/pull/122

**Pull request merged**